apisix/plugins/proxy-rewrite.lua

```diff
@@ -153,10 +154,20 @@ function _M.rewrite(conf, ctx)
         end
     end
 
-    upstream_uri = core.utils.uri_safe_encode(upstream_uri)
+    local index, _ = find_str(upstream_uri, "?")
```

Comment:
Should use plain text style `string.find`.

Reply:
> Should use plain text style `string.find`.

Edited.